enkoder/enkoder.ino

```diff
@@ -0,0 +1,67 @@
+#include <Encoder.h>
+
+Encoder Ludi(2,3);
+int speedpin=7;
+int directionpin1=6;
+int directionpin2=5;
+
+long pozicijaLudi  = -999, noviLudi;
+int start=0, f=1;
+float ugao, jedinicni_ugao=(float)360/2400,ugaona_brzina, volti=6*21.25;
+
+void setup() {
+  Serial.begin(115200);
+
+  pinMode(speedpin,OUTPUT);
+  pinMode(directionpin1,OUTPUT);x 
+  pinMode(directionpin2,OUTPUT);
+  analogWrite(speedpin,127);  //na motoru koji smo koristili 21.25 je otprilike 1 volt
+  digitalWrite(directionpin2,HIGH);
+  digitalWrite(directionpin1,LOW);
+}
+
+void reset (Encoder Ludi){
+  if (Serial.available()) {
+    Serial.read();
+    Serial.println("Reset");
+    Ludi.write(0);
+  }
+}
+
+void stampaj(float x){
+  Serial.print(x);
+}
+
+float merenje_ugla(Encoder Ludi){
+  noviLudi = Ludi.read();
```

Comment:
Hajde stavite normalna imena promenljiva pa da možemo da pričamo. Ne možete da očekujete da razumem liniju `noviLudi = Ludi.read()`. Ne znam koja je jedinica, kako se to mapira na jednačine koje ste pisali ranije i slično.